specs/paymentrequest.html

```diff
@@ -660,17 +660,26 @@ <h2 id="state-transitions">State transitions</h2>
         The following fields MAY be passed to the <a><code>PaymentRequest</code></a> constructor:
       </p>
       <dl>
-        <dt><code><dfn>requestShipping</dfn></code></dt>
+        <dt><code>requestedData</code></dt>
         <dd>
-          This <em>boolean</em> value indicates whether the <a>user agent</a> should collect and return
-          a shipping address as part of the payment request. For example, this would be set to
-          <code>true</code> when physical goods need to be shipped by the merchant to the user.
-          This would be set to <code>false</code> for an online-only electronic purchase transaction.
-          If this value is not supplied then the the <a><code>PaymentRequest</code></a> behaves as
-          if a value of <code>false</code> had been supplied.
+          A sequence containing one or more <code>RequestedData</code> values that the user agent should collect and return as part of the payment request. For example, if a merchant needs to collect shipping address and email, they can request both <code>shipping-address</code> and <code>email</code>
         </dd>
       </dl>
     </section>
+    
+    <section>
+      <h2>RequestedData Registry</h2>
+      <pre class="idl">
+        enum RequestedData {
+          "shipping-address",
+          "email",
```

Comment:
Ok, I'm used to this sort of stuff being hashed out in an issue before there is a PR. Not sure what this group's process is...